hamburger_menu.js

```diff
@@ -0,0 +1,7 @@
+
+var toggleButton = document.getElementsByClassName('toggle-button')[0];
+var navLink = document.getElementsByClassName('nav-link')[0];
+toggleButton.addEventListener('click',showMenu);
+function showMenu(){
+    navLink.classList.toggle("active");
+}
```

Comment:
although not in the scope of this homework the hamburger menu using is working well - we will revisit wrking with the DOM in JavaScript later in the JS modules :)

Reply:
Thank you for your comment.